core/src/test/scala/akka/persistence/cassandra/CassandraLifecycle.scala

```diff
@@ -99,14 +103,11 @@ trait CassandraLifecycle extends BeforeAndAfterAll with TestKitBase {
 
   def systemName: String
 
-  def port(): Int = 9042
+  lazy val cluster: CqlSession =
+    Await.result(session.underlying(), 10.seconds)
 
-  lazy val cluster = {
-    CqlSession
-      .builder()
-      .withLocalDatacenter("datacenter1")
-      .addContactPoint(new InetSocketAddress("localhost", port()))
-      .build()
+  def session: CassandraSession = {
+    CassandraSessionRegistry(system).sessionFor("akka.persistence.cassandra")
   }
```

Comment:
This was one of the test changes. Using the "right" session instead of an add-hoc new CqlSession (which wouldn't have the right config for connecting to cloud)

Reply:
also then need to remove the `close` of it in cleanup

Reply:
fixed, two places